spec/services/dates/review_weekly_selection_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Dates::ReviewWeeklySelection do
+  subject(:service) do
+    described_class.new(
+      vrn: vrn,
+      zone_id: zone_id,
```

Comment:
```suggestion
      zone_id: @uuid,
```
and remove let's for vrn and zone_id, try do not use `let` if you dont use it in tests